### ISSUE TITLE
ipywidgets fails after installing jupyter_contrib_nbextensions

### ISSUE THREAD

**Issue by josephcslater** [ipython-contrib/jupyter_contrib_nbextensions#1261]
I've been having issues with reliability with ipywidgets. A clean install of anaconda (5.1) and my notebook widgets work. Installing only jupyter_contrib_nbextensions next- the widgets stop working (and don't show). 

Is there a solution for this? I love both sets of toys! (MacOS X High Sierra).

**Comment by josephcslater** [comment]
After having this problem for months, things are now working, and I have no idea why. So- I'll embarrassingly close my own issue- until it comes up again.

**Comment by josephcslater** [comment]
I've been coping with ipywidgets going dead for two years now. I don't use them daily, so when they die, I have perhaps hundreds of upgrades in the meantime. I tried to isolate last night. Thought I had it. Then they started working right after I posted. I haven't had them work on a non-fresh conda install. So, now I'm back to hoping to figure it out someday in the future. I need to post on their issues as it seems  jupyter_contrib_nbextensions  isn't the cause. Or maybe is it when I turn one of them on. I haven't a clue. My machine seems to be an entirely special case of schizophrenia.
